save.go

```diff
@@ -46,6 +47,15 @@ func saveImage(image imgutil.Image, additionalNames []string, logger Logger) (Im
 	default:
 	}
 
+	manifestSize, sizeErr := image.ManifestSize()
+	if sizeErr != nil {
+		return ImageReport{}, sizeErr
+	}
+	if manifestSize != 0 {
+		imageReport.ManifestSize = strconv.FormatInt(manifestSize, 10)
+		logger.Debugf("\n*** Manifest Size: %d\n", manifestSize)
+	}
```

Comment:
I don't see that we have different logic depending on whether the platform API is 0.6+. Rather than pass api-related logic into `saveImage`, maybe we could have the exporter / rebaser do some post-processing to strip this info out of the report for lower platform APIs. What do you think?

Reply:
Similarly to the above, we should strip manifest size from the report for daemon builds.

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/498/commits/fbf519cd2b6d74906571808c3207936d56eb4757